File: CS/fGrid11/Person.cs
using System;
using System.Collections.ObjectModel;
using System.Linq;

namespace fGrid11
{
    public class Person
    {
        public Person(int i) {
            LastName = "LastName #" + i;
            Age = i * 10;
        }
        public static ObservableCollection<Person> CreateList {
            get {
                ObservableCollection<Person> list = new ObservableCollection<Person>();
                for (int i = 0; i < 10; i++)
                    list.Add(new Person(i) { NameFontSize = 12 + (i % 6) });
                return list;
            }
        }
        public int Age { get; set; }
        public string LastName { get; set; }
        public double NameFontSize { get; set; }
    }
}

File: VB/fGrid11/Person.vb
Imports System
Imports System.Collections.ObjectModel
Imports System.Linq

Namespace fGrid11
	Public Class Person
		Public Sub New(ByVal i As Integer)
			LastName = "LastName #" & i
			Age = i * 10
		End Sub
		Public Shared ReadOnly Property CreateList() As ObservableCollection(Of Person)
			Get
				Dim list As New ObservableCollection(Of Person)()
				For i As Integer = 0 To 9
					list.Add(New Person(i) With {.NameFontSize = 12 + (i Mod 6)})
				Next i
				Return list
			End Get
		End Property
		Public Property Age() As Integer
		Public Property LastName() As String
		Public Property NameFontSize() As Double
	End Class
End Namespace
